entity-services/src/main/xdmp/entity-services/entity-services-impl.xqy

```diff
@@ -140,6 +140,26 @@ declare private variable $esi:model-schematron :=
         <iso:rule context="es:word-lexicon">
          <iso:assert test="string(.) = (../es:properties/*/local-name(.))">Word lexicon property <xsl:value-of select="." /> doesn't exist.</iso:assert>
         </iso:rule>
+        <iso:rule context="namespace">
+         <iso:assert test="matches(., '^[a-z]+:')">Namespace property must be a valid absolute URI.  Value is <xsl:value-of select="." />.</iso:assert>
+         <iso:assert test="../namespacePrefix">If a type has a namespace property, it must also have a namespacePrefix property.</iso:assert>
+        </iso:rule>
+        <iso:rule context="es:namespace">
+         <iso:assert test="matches(., '^[a-z]+:')">Namespace property must be a valid absolute URI.  Value is <xsl:value-of select="." />.</iso:assert>
+         <iso:assert test="../es:namespace-prefix">If a type has a namespace property, it must also have a namespace-prefix property.</iso:assert>
+        </iso:rule>
```

Comment:
namespace and es:namespace. Do we have it so that one is for json and other for xml?

Reply:
That's right.  I made two different rules because of the slightly different messages.  I couldn't decide whether to put more mental effort into combining them.